server/routes/api.js

```diff
@@ -0,0 +1,60 @@
+const express = require("express");
+const router = express.Router();
+const nbaUrl = "http://data.nba.net/10s/prod/v1/2018/players.json";
+const request = require("request");
+
+const teamToIDs = {
+  lakers: "1610612747",
+  warriors: "1610612744",
+  heat: "1610612748",
+  suns: "1610612756"
+};
+const dreamTeam = [];
+
+router.get("/teams/:teamName", function(req, res) {
+  const teamName = req.params.teamName;
+  const id = teamToIDs[teamName];
+  request(nbaUrl, function(error, result) {
+    const jsonRes = JSON.parse(result.body).league.standard;
+    const players = jsonRes
+      .filter(r => {
+        if (r.teamId === id && r.isActive) {
+          return r;
+        }
+      })
+      .map((player, i) => {
+        return {
+          fName: player.firstName,
```

Comment:
You should use clear and understandable names for classes, functions, and variables (for example: firstName etc.)